src/main/java/com/epam/ta/reportportal/core/user/impl/DeleteUserHandler.java

```diff
@@ -79,6 +84,8 @@ public OperationCompletionRS deleteUser(String userId, String principal) {
 			throw new ReportPortalException("Error while deleting user", exp);
 		}
 
+		logIndexer.deleteIndex(user.getLogin().toLowerCase()+PersonalProjectService.PERSONAL_PROJECT_POSTFIX.toLowerCase());
```

Comment:
please, install and use our project formatter from [here](https://github.com/reportportal/reportportal/blob/master/idea_formatting_profile.xml)